### ISSUE TITLE
Switch data file date logic to UTC

### ISSUE THREAD

**Issue by pastly** [pastly/simple-bw-scanner#113]

**Comment by pastly** [comment]
I this this primarily entails switching `date.fromtimestamp` to `datetime.utcfromtimestamp` in all the following places and watching what breaks.

```
(venv-editable) cranium:simple-bw-scanner mtraudt$ find sbws tests -type f -name '*.py' | xargs grep fromtimestamp
sbws/core/cleanup.py:    today = datetime.utcfromtimestamp(time.time())
sbws/core/stats.py:    first = date.fromtimestamp(first_time)
sbws/core/stats.py:    last = date.fromtimestamp(last_time)
sbws/lib/resultdump.py:    today = date.fromtimestamp(time_now())
sbws/lib/resultdump.py:    dt = date.fromtimestamp(result.time)
tests/core/test_stats.py:            tmpdir, 'datadir', date.fromtimestamp(time.time())),
tests/core/test_stats.py:            tmpdir, 'datadir', date.fromtimestamp(time.time())),
```

**Comment by juga0** [comment]
I've not reviewed how you were generating those timestamps, so it might be fine here, but to avoid problems with timezones, we might need to use tzinfo (https://docs.python.org/3/library/datetime.html#datetime.datetime.tzinfo)

**Comment by pastly** [comment]
In sbws I've been strongly preferring the use of raw unix timestamps with no parsing into date/datetime structures. Unix timestamps are always in UTC. 

As I learned right before opening this ticket, `date.fromtimestamp` seems to convert the timestamp into your local time. Thus in this ticket I'm going to replace instances of that with `datetime.utcfromtimestamp`.

I hope sbws never has to think about timezones. In fact, I've taken a few steps in time-dependent things to avoid having to think about timezones. Off the top of my head: I made `sbws generate` load 1 extra data file in the past just in case.
